ystia/yorc
# Feature/gh 425 expose bypass error

# Pull Request description

As: An integrator
I want to: Have choice to bypass errors or not on workflows
So that: I can stop on errors for an undeploy workflow

AC1: Should be part of the REST API
AC2: Think if it should be limited to undeploy workflow
AC3: Default behavior should remain unchanged
AC4: Check that a4c plugin handle this case for undeployments
AC5: Sounds a good opportunity for increasing our test coverage on rest package

## Description of the change

### What I did
- Already exposed for custom workflows
- Add "stopOnError" query parameter handling to delete handler and store it via task data "continueOnError"
- Allow the worker to handle "continueOnError" on undeploy workflow
- Add option stop-on-error for CLI to use it

### How to verify it
- Deploy the welcome application
- Stop the python web server via custom command (this will make the undeploy firtst step fail)
- Undeploy it with CLI or REST client with stopOnError param
- Check the deployment status is UNDEPLOYMENT_FAILED and the undeployment is stopped

### Description for the changelog
* Expose bypass error parameter on workflow ([GH-425](https://github.com/ystia/yorc/issues/425))
## Applicable Issues
#425
